clonelists/Sony - PlayStation (Redump).json

```diff
@@ -6930,6 +6946,9 @@
 			"titles": [
 				{"searchTerm": "Wanted"},
 				{"searchTerm": "Hot Shot", "priority": 2}
+			],

```

Comment:
Nice catch on the compilation :) After watching a [gameplay video](https://www.youtube.com/watch?v=XIbDyHpbWGc&ab_channel=Smiggiles) online, there are a few more places this can be added:

* ~Wanted~ (Already added, called Wild West in the compilation)
* ~Hot Shot~ (Already added, called Ace in the compilation)
* Silent Iron (called War Drones in the compilation)
* Space Rider (called Space Rebellion in the compilation)
* Omega Assault (called GSquad in the compilation)
* Jet Ace (called IF22 in the compilation)
* Flying Squadron (called Air Rescue in the compilation)